### DISCUSSION TITLE
euro when adding filter

### DISCUSSION THREAD

**Discussion by newminder** [YetiForceCompany/YetiForceCRM#16913] [Ignored/Incomplete]
when trying to add filter, the error show up. 
![1676824323383](https://user-images.githubusercontent.com/18006681/219961433-9e1928d6-a64d-43d6-9a37-f846974d15d5.jpg)

**Comment by mariuszkrzaczkowski** [comment]
bug/error >> https://github.com/YetiForceCompany/YetiForceCRM/issues/new?assignees=&labels=&template=1-bug-report.md&title=%5Bbug%5D+
question >> https://github.com/YetiForceCompany/YetiForceCRM/discussions

it is important that there is a complete set of information, what you send means nothing

**Reply by mariuszkrzaczkowski** [reply]
https://github.com/YetiForceCompany/YetiForceCRM/issues/13908

**Reply by newminder** [reply]
thank you,  already fixed it.

**Reply by mariuszkrzaczkowski** [reply]
share the solution with others
